tests/static_map/insert_and_find_test.cu

```diff
@@ -23,57 +23,16 @@
 #include <thrust/device_vector.h>
 #include <thrust/execution_policy.h>
 #include <thrust/functional.h>
-#include <thrust/sequence.h>
+#include <thrust/iterator/counting_iterator.h>
+#include <thrust/iterator/transform_iterator.h>
+
+#include <cuda.h>
```

Comment:
do we need this header?

Reply:
cuda.h is not requried. I missed this during cleanup.